### PR TITLE
Changed shape of button/color is blue when hovered

### PR DESCRIPTION
**Pull request by Subrina45**
Willing to make changes when needed.

### CODE REVIEW

File: src/index.css [hunk 1]
```diff
@@ -59,7 +70,7 @@ p {
     float: right;
     height: 2vw;
     box-sizing: border-box;
-    background-color: white;
+    background-color: rgb(188, 240, 200);
```
Comment (AnushaTiwari5) [inline src/index.css:73]: this feels too bright, considering the rest of our palette are dim pastels. change it to rgb(190, 215, 190) instead

**Review by AnushaTiwari5 — CHANGES_REQUESTED** [review]

**Review by DawnTheWitch — COMMENTED** [review]
I thought he wanted the button to be pressed in when it's currently active, not just when you press it but maybe that's just how I read into it

File: src/index.css
Comment (AnushaTiwari5) [inline src/index.css:1]: Maybe look into making the title font size bigger/bold? It doesn't really stand out right now

File: src/index.css [hunk 2]
```diff
@@ -59,7 +72,7 @@ p {
     float: right;
     height: 2vw;
     box-sizing: border-box;
-    background-color: white;
+    background-color: rgb(190, 215, 190);
```
Comment (James-Oswald) [inline src/index.css:75]: Again, colors should be done using exclusively hex rgb values (prefixed with #) or the rgb(), either is fine but you should not mix styles, pick one and be consistent.

**Review by James-Oswald — CHANGES_REQUESTED** [review]

**Pull request merged**